### DISCUSSION TITLE
Community meeting on 12th September

### DISCUSSION THREAD

**Discussion by pavithraes** [nebari-dev/nebari#1995] [Community meetings]
We'll have a community meeting tomorrow, please add discussion topics and updates to the meeting notes!

🕒 Meeting date & time: 2-3 pm GMT

💻 Meeting room: [meet.google.com/rio-ngrj-kyh](https://meet.google.com/rio-ngrj-kyh)

📝 Meeting notes: https://hackmd.io/@nebari/rktMu63Cn

🗓️ Meeting calendar: [nebari.dev/docs/community#community-meetings](https://www.nebari.dev/docs/community#community-meetings)

**Comment by mhmohona** [comment]
Hello, in [calendar](https://www.nebari.dev/docs/community/#community-meetings) I am seeing the meeting time is at 1.30PM GMT but here I am seeing it's 2PM GMT.

![image](https://github.com/nebari-dev/nebari/assets/14244685/2863cc3f-4e4d-4b61-93c7-96f26fd61f2f)

7.30PM is 1.30PM GMT in my local time.

**Reply by pavithraes** [reply]
Thanks for sharing. The embedded calendar does seem wrong -- I'll look into it. 

But for your reference, the meeting time is 2pm GMT on alternate Tuesdays. :)
